cantools/database/can/signal.py

```diff
@@ -191,8 +193,8 @@ def __init__(self,
                  length: int,
```

Comment:
TODO for another day: we should make this constructor (and others like it, erhm `Message`) keyword-only: instantiating an object with a dozen or so positional parameters is asking for trouble IMO...